src/appleseed/renderer/modeling/bsdf/bsdfsample.h

```diff
@@ -61,6 +61,8 @@ class BSDFSample
     // Roughness.
     float                           m_max_roughness;        // BSDF roughness
```

Comment:
Unrelated to your PR, but this should be moved to the right section as well. Maybe you can do it in this PR, in a separate commit?